ros_buildfarm/templates/release/deb/binarypkg_job.xml.em

```diff
@@ -102,7 +102,7 @@ but disabled since the package is blacklisted (or not whitelisted) in the config
         ' ' + os_code_name +
         ' ' + arch +
         ' ' + ' '.join(repository_args) +
-        ' --binarypkg-dir $WORKSPACE/binarydeb' +
+        ' --binarypkg-dir /tmp/binarydeb' +
```

Comment:
This argument is currently unused. After the job is changed to pass the mount point, we can change the Dockerfile to expect the directory to be mounted there.